simulation-construction-set-examples/falling-brick/src/main/java/us/ihmc/fallingBrick/FallingBrickDefinition.java

```diff
@@ -0,0 +1,122 @@
+package us.ihmc.fallingBrick;
+
+import us.ihmc.euclid.tuple3D.Vector3D;
+import us.ihmc.scs2.definition.collision.CollisionShapeDefinition;
+import us.ihmc.scs2.definition.geometry.Box3DDefinition;
+import us.ihmc.scs2.definition.geometry.GeometryDefinition;
+import us.ihmc.scs2.definition.robot.GroundContactPointDefinition;
+import us.ihmc.scs2.definition.robot.RigidBodyDefinition;
+import us.ihmc.scs2.definition.robot.RobotDefinition;
+import us.ihmc.scs2.definition.robot.SixDoFJointDefinition;
+import us.ihmc.scs2.definition.visual.ColorDefinitions;
+import us.ihmc.scs2.definition.visual.MaterialDefinition;
+import us.ihmc.scs2.definition.visual.VisualDefinition;
+
+public class FallingBrickDefinition extends RobotDefinition
+{
+   private static final String FALLINGBRICK = "fallingBrick";
+
+   // Define the parameters of the brick
+   private static final double BASE_H = 0.1, BASE_W = 0.2, BASE_L = 0.3;
+   private static final double M1 = 2.0;
+   private static final double Ixx1 = (1.0 / 12.0) * M1 * (BASE_W * BASE_W + BASE_H * BASE_H), Iyy1 = (1.0 / 12.0) * M1 * (BASE_L * BASE_L + BASE_H * BASE_H),
+         Izz1 = (1.0 / 12.0) * M1 * (BASE_L * BASE_L + BASE_W * BASE_W);
+
+   public FallingBrickDefinition()
+   {
+      // Call parent class "Robot" constructor. The string "FallingBrick" will be the name of the robot.
+      super(FALLINGBRICK); // creates an instance of the class Robot named "FallingBrick" in the SCS system.
+
+      // Create the top (fixed) link that serves as the base of the brick
+      // Define this link as the root body of our robot
+      RigidBodyDefinition elevator = new RigidBodyDefinition("elevator");
+      setRootBodyDefinition(elevator);
+
+      // Setup the brick body
+      RigidBodyDefinition brick = createBrickRigidBody();
+      brick.setMass(M1);
+      brick.getMomentOfInertia().setToDiagonal(Ixx1, Iyy1, Izz1);
+      brick.setCenterOfMassOffset(0.0, 0.0, 0.0);
+
+      // Define and add a floating joint to the brick
+      SixDoFJointDefinition floatingJoint = new SixDoFJointDefinition(getRootJointName());
+      // Connect this joint to the robot base
+      elevator.addChildJoint(floatingJoint);
+      floatingJoint.setSuccessor(brick);
+
+      // Add ground contact points to the brick
+      /*
+       * The first parameter is the name of the ground contact point. The second parameter is the offset
+       * vector with respect to the joint the contact point will be attached to.
```

Comment:
This doc needs updating:
Each GroundContactPointDefinition will be used to create a GroundContactPoint attached to the robot. The ground contact points are used by with the contact point based physics engine to prevent penetration with the environment.

Reply:
Added to the description.